src/MSALWrapper.Test/AuthFlow/BrokerTest.cs

```diff
@@ -94,7 +94,7 @@ public async Task BrokerAuthFlow_GetTokenSilent_ReturnsNull()
             this.pcaWrapperMock.VerifyAll();
             authFlowResult.TokenResult.Should().Be(null);
             authFlowResult.Errors.Should().BeEmpty();
-            authFlowResult.AuthFlowName.Should().Be("Broker");
+            authFlowResult.AuthFlowName.Should().Be("broker");

```

Comment:
Wondering if the friendly names will break any of my Kusto queries. For any auth flow specific query, I do the following -
```
AppEvents
| where Name startswith "azureauth_authflow"
```

This should be okay since the commands should still start with `azureauth_authflow`.

Reply:
Yes those queries won't be impacted - but the others will. We already have a feature on the backlog to go "re-up" all the Workbooks because the queries are already broken.

Reply:
Which ones would break as my service queries could break too, so I wanted to be aligned. :)

Reply:
I don't know off hand which ones are broken, but those impacted I expect to be those using the Name explicitly and queries looking at error_messages, since we've changed how the exceptions are logged. I don't think the aggregate queries look at those, and we can update the aggregates to exclude versions before 0.6.0 which is when the errors collection change was released.